src/keyfile.c

```diff
@@ -86,12 +86,10 @@
 #define GEANY_DEFAULT_TOOLS_BROWSER		"open -a safari"
 #define GEANY_DEFAULT_FONT_SYMBOL_LIST	"Helvetica Medium 12"
 #define GEANY_DEFAULT_FONT_MSG_WINDOW	"Menlo Medium 12"
-#define GEANY_DEFAULT_FONT_EDITOR		"Menlo Medium 12"
```

Comment:
Specifically, I think it would require a file called `org.geany.Settings.gschema.override` in the bundle's equivalent of the `$(datadir)/glib-2.0/schemas/` directory with the following contents:

```ini
[org.geany.Settings]
editor-font="Menlo Medium 12"
```